code/modules/clothing/gloves/color.dm

```diff
@@ -17,8 +17,11 @@
 	if((slot == ITEM_SLOT_GLOVES) && (user.mind?.assigned_role in GLOB.security_positions))
 		SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "sec_black_gloves", /datum/mood_event/sec_black_gloves)
 
-/obj/item/clothing/gloves/color/black/dropped(mob/user)
+/obj/item/clothing/gloves/color/black/dropped(mob/living/carbon/user)
 	..()
+	var/mob/living/carbon/C = user
```

Comment:
user is already assumed to be of the type mob/living/carbon due to param changes, why copy it again?

also why does dropped get called on a pair of gloves that is not src?

Reply:
Yeah but if the item being dropped is not src then there's a problem

Reply:
There... is, go host a local game, join as assistant, get two pairs of insuls, equip one, drop and pick up the other. You'll see that your positive moodlet dissapears. I've sent the video to you in Discord.

Reply:
With this fix, you have to move gloves to your hands to drop them, making the moodlet never be removed

Reply:
That ain't true though, look at the videos in testing evidence.